website/auth.py

```diff
@@ -160,6 +160,15 @@ def forget_current_user():
     session.pop("profile_image", None)  # Delete profile image id if existing
 
 
+def show_roles_of_variables():
+    """Whether the roles of variables should be shown in the variable view.
+    (If not, only variable names and values will be shown in the list).
+    Checks the configuration in the environment variable $SHOW_ROLES    
+    """
+    can_show_roles = os.getenv("SHOW_ROLES")
```

Comment:
If `SHOW_ROLES` is not set you can also add a default value:

```suggestion
    can_show_roles = os.getenv("SHOW_ROLES", False)
```

Reply:
Oh I just noticed this is in `auth.py` can you move it to `utils.py`?